packages/styling/src/styles/theme.ts

```diff
@@ -107,12 +107,20 @@ export function createTheme(theme: IPartialTheme, depComments: boolean = false):
     ...theme.semanticColors
   };
 
+  let defaultFontStyles: IFontStyles = DefaultFontStyles;
+  if (theme.defaultFontStyle) {
+    for (const fontStyle of Object.keys(DefaultFontStyles)) {
+      defaultFontStyles[fontStyle] = { ...defaultFontStyles[fontStyle], ...theme.defaultFontStyle };
+    }
+  }
+
   return {
     palette: newPalette,
     fonts: {
-      ...DefaultFontStyles,
+      ...defaultFontStyles,
       ...theme.fonts
```

Comment:
I think this will be a shallow mix:

defaultFontStyles:
{
  small: { bg: red, size: 10 }
}

theme.fonts:
{
  small: { bg: blue }
}

Result will be missing size: 10

Reply:
(you may need to use Object merge.)

Reply:
You're right, I'll look into merge!

Reply:
Should hopefully be fixed! Added a test for this scenario as well.